pom.xml

```diff
@@ -2273,7 +2273,7 @@
             </activation>
             <properties>
                 <modular.jdk.args>
-	            --add-opens=java.base/java.security=ALL-UNNAMED
+	            --add-opens=java.base/java.security=ALL-UNNAMED --add-opens=java.base/sun.security.ssl=ALL-UNNAMED
```

Comment:
Can this be a separate line?

Perhaps and xml comment above too that explains what it's there.

But the separate line would improve the odds that future git annotate users will be able to understand why it is there.